packages/shopify-app-remix/package.json

```diff
@@ -12,15 +12,18 @@
   "homepage": "https://github.com/Shopify/shopify-app-js/tree/main/packages/shopify-app-remix",
   "author": "Shopify Inc.",
   "license": "MIT",
-  "main": "./build/cjs/index.js",
-  "types": "./build/ts/index.d.ts",
+  "types": "./build/ts/server/index.d.ts",
+  "main": "./build/cjs/server/index.js",
   "exports": {
     ".": {
-      "import": "./build/cjs/index.js",
-      "require": "./build/cjs/index.js",
-      "types": "./build/ts/index.d.ts"
+      "types": "./build/ts/server/index.d.ts",
+      "default": "./build/cjs/server/index.js"
     },
-    "./adapters/*": "./build/cjs/adapters/*/index.js"
+    "./adapters/*": "./build/cjs/adapters/*/index.js",
```

Comment:
Should adaptors be provided by the server export and aliased?

Reply:
I'd rather keep them fully separate because that makes it harder to accidentally import them and break apps.